src/plugins/discover/public/application/components/sidebar/discover_field_bucket.tsx

```diff
@@ -45,13 +53,29 @@ export function DiscoverFieldBucket({ field, bucket, onAddFilter }: Props) {
     }
   );
 
+  const tooltipContent = `${bucket.value}: ${bucket.count} (${bucket.percent}%)`;
+
   return (
     <>
-      <EuiFlexGroup gutterSize="xs" responsive={false}>
-        <EuiFlexItem className="eui-textTruncate">
-          <EuiText size="xs" className="eui-textTruncate">
-            {bucket.display === '' ? emptyTxt : bucket.display}
-          </EuiText>
+      <EuiFlexGroup justifyContent="spaceBetween" responsive={false} gutterSize="s">
+        <EuiFlexItem className="dscFieldDetails__barContainer" grow={1}>
+          <EuiToolTip content={tooltipContent} delay="long" position="right">
+            <>
+              <EuiFlexGroup justifyContent="spaceBetween" gutterSize="xs" responsive={false}>
+                <EuiFlexItem grow={1} className="eui-textTruncate">
+                  <EuiText size="xs" className="eui-textTruncate">
+                    {bucket.display === '' ? emptyTxt : bucket.display}
+                  </EuiText>
```

Comment:
By removing the EuiTooltip moving it's text to `StringFieldProgressBar`, the functional test should work, no performance troubles, and I think for long fields we should just add a `title` attribute in the EuiText, like in `discover_field`

Reply:
@kertal @cchaos Before we were using the tooltip to show `count` which we're now not showing anywhere else though. So I'm guessing maybe it wasn't just an a11y thing. Are we ok with not showing `count` anymore?

Reply:
I'm fine with not showing it the way Discover did it before wasn't really helpful and also Lens doesn't display the count.

<img width="266" alt="Bildschirmfoto 2020-06-25 um 17 47 25" src="https://user-images.githubusercontent.com/463851/85753589-adb48180-b70c-11ea-9ebd-eaa5c199b882.png">

 I think the component main purpose is to show the top values of a field,  and a visual chart + percentage is more helpful the providing a exact count, which doesn't matter in my opinion.

Reply:
The only worry I have about completely removing the count is that it's really hard to **take away** a feature that already exists. There may be users who rely and use this feature a lot and would report bugs if we just remove it. Lens doesn't include it mainly because this was overlooked.

Lets go ahead and keep the original format from the tooltip as @andreadelrio had it in an earlier commit `field value: count (%)`, but inside of the `title` prop for now and we can revisit the tooltip later when we start moving these into popovers.